postgres/init/22-portal-content.sql

```diff
@@ -271,6 +271,7 @@ f032a047-d035-11ec-9d64-0242ac120002	not yet known-3
 f032a048-d035-11ec-9d64-0242ac120002	not yet known-4
 789e31ee-0fdb-4e42-a819-23e0cfb1179b	https://catenax-dt-rec.authentication.eu10.hana.ondemand.com
 cf207afb-d213-4c33-becc-0cabeef174a7	https://catenax-int-dismantler-s66pftcc.authentication.eu10.hana.ondemand.com
+f032a045-d035-11ec-9d64-0242ac120002	technical_roles_management
```

Comment:
That line results in:
2022-07-26 11:58:11.662 UTC [73] ERROR:  duplicate key value violates unique constraint "pk_iam_clients"
2022-07-26 11:58:11.662 UTC [73] DETAIL:  Key (id)=(f032a045-d035-11ec-9d64-0242ac120002) already exists.
2022-07-26 11:58:11.662 UTC [73] CONTEXT:  COPY iam_clients, line 19
2022-07-26 11:58:11.662 UTC [73] STATEMENT:  COPY portal.iam_clients (id, client_client_id) FROM stdin;
psql:/docker-entrypoint-initdb.d/22-portal-content.sql:275: ERROR:  duplicate key value violates unique constraint "pk_iam_clients"
DETAIL:  Key (id)=(f032a045-d035-11ec-9d64-0242ac120002) already exists.
CONTEXT:  COPY iam_clients, line 19

Reply:
I have already run that in int and dev and was now really confused why it should not work in devenv.
But I see that devenv has a different data basis as int and dev istelf, since we are not able to bring the data with a delta load into dev.demo or int.
I will change this in the afternoon.

Reply:
fixed